<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_record_boxing</class>
 <widget class="QDialog" name="Dlg_record_boxing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>波形回顾</string>
  </property>
  <widget class="QPushButton" name="m_btn_forwd_fast">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>410</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_forwd_end">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>410</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back_end">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>|&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_return">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back_fast">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_back">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 10, 10);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_sprate_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>temp1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_beatrate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>spire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nibp_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>nibp</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nibp_ss">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>temp2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sprate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_spo2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_nibp_sz">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_temp1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>CO2 Et</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nibp_pj">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_nibp_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>beatrate:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nibp_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>spo2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pulserate1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>pulserate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pulserate">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(249, 26, 12);</string>
    </property>
    <property name="text">
     <string>---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="m_cmbb_wave">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>456</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形类型</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
